Added Building on Windows section to Developer Documentation (#11)
Bit-Quill/tinkerpop: docs/src/dev/developer/development-environment.asciidoc

NOTE: For those using Windows, efforts have been made to keep the build OS independent, but, in practice, it is likely
that TinkerPop's build system will only allow for a minimum build at best. +
 +

>  Refer to <<building-on-windows>> section for more details.

Author: ```suggestion
 Refer to <<building-on-windows>> section for more details.
 
```

>  Refer to <<building-on-windows>> section for more details.

Author: added in [494c846](https://github.com/Bit-Quill/tinkerpop/pull/11/commits/494c846afc36f03fbe154654ee375c3d0ca85594)

** Generate web site locally: `bin/generate-home.sh`
** Publish web site: `bin/publish-home.sh <username>`

[[building-on-windows]]
== Building On Windows

> The following are the steps a Windows user must take in order to Build Tinkerpop on Windows:

Reviewer: Phrasing edits. 
```suggestion
The following steps must be taken in order to build TinkerPop on Windows:
```

. Install winutils for Hadoop
* Download the link:https://github.com/cdarlint/winutils/blob/master/hadoop-2.7.7/bin/winutils.exe[winutils.exe] file for Hadoop version `2.7.7` from cdarlint/winutils
* Place `winutils.exe` file wherever you would like to in the following folder structure:
** `hadoop-2.7.7/bin/winutils.exe`

> * Set `HADOOP_HOME` to point to `hadoop-2.7.7` folder

Reviewer 2: Does it make sense to have this part be more generic rather than to refer to Hadoop version 2.7.7? Maybe just tell them to grab the required hadoop version for their branch?

Author: the Hadoop version is fixed in `3.5` to `2.7.7`

Reviewer 3: may be better to push different text for 3.5 and 3.6?

. Run `mvn clean install` from root of tinkerpop
. Follow IDE specific steps if applicable:
* <<intellij>>

You should now be able to work with Tinkerpop on Windows.

> NOTE: The Windows build of Tinkerpop has currently only been tested to stably work on v3.5. Any other versions of Tinkerpop on Windows may still build, but to a partial extent.

Reviewer: ```suggestion
NOTE: Currently, the Windows build of TinkerPop is only stable on versions 3.5.x. Other versions of TinkerPop on Windows may still build, but to a partial extent.
```
